package.json

```diff
@@ -244,6 +245,7 @@
     "react-responsive-masonry": "^2.1.7",
     "react-router-dom": "^6.4.2",
     "react-scrollbars-custom": "^4.1.1",
+    "react-select": "^5.8.0",
```

Comment:
Why is this needed, don't we already have a select component ?

Unless there's a good reason for having it, I'm not ok with it, as it adds 700kb to the bundle

Reply:
This lib has implemented a lot of features related to accessibility - you can easily navigate between the options using a keyboard, open the select using it, etc. It shows the menu in a place where there's enough space for it, for example, if you place a dropdown at the end of the page, then the menu would appear on the top of the dropdown and not on the bottom causing an overflow. It's also generic, so it's pretty easy to use it everywhere because you can create a custom option and pass it to the component without any weird if statements in the code. It's also a lot easier to modify than the component that we had in the app. You can also use it with `react-hook-form` lib.

Also, it's only about 30kb - https://bundlephobia.com/package/react-select@5.8.0